api/v1/models/labels.go

```diff
@@ -25,3 +27,22 @@ func (m Labels) Validate(formats strfmt.Registry) error {
 	}
 	return nil
 }
+
+// GetMap returns a map with the Labels in key-value format. If one label
```

Comment:
This file is autogenerated, I'm afraid. We'll have to add this code somewhere else.